jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/util/GoTimeUtilTest.java

```diff
@@ -16,23 +16,22 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
 public class GoTimeUtilTest {
 
     @Test
     public void testConversion() {
-        Assert.assertEquals(new Integer(23), GoTimeUtil.durationSeconds("23s"));
-        Assert.assertEquals(new Integer(0), GoTimeUtil.durationSeconds("0.5s"));
-        Assert.assertEquals(new Integer(0), GoTimeUtil.durationSeconds("3ms"));
-        Assert.assertEquals(new Integer(0), GoTimeUtil.durationSeconds("3ns"));
-        Assert.assertEquals(new Integer(1), GoTimeUtil.durationSeconds("1002ms"));
-        Assert.assertEquals(new Integer(123), GoTimeUtil.durationSeconds("2m3s"));
-        Assert.assertEquals(new Integer(3663), GoTimeUtil.durationSeconds("1h1m3s"));
-        Assert.assertEquals(new Integer(1810), GoTimeUtil.durationSeconds("0.5h0.1m4s"));
-        Assert.assertEquals(new Integer(-15), GoTimeUtil.durationSeconds("-15s"));
-        Assert.assertEquals(new Integer(30), GoTimeUtil.durationSeconds("2h-119.5m"));
+        Assert.assertEquals(Integer.valueOf(23), GoTimeUtil.durationSeconds("23s"));
```

Comment:
Why are we boxing the integers?

Reply:
I see now why the boxing/unboxing.

We need to get our code cleaner.
Possible options:
- Change method so that it returns an `Optional<Integer>` (although you'll still get a minor Sonar warning in the test)
- Use assertj fluent assertions (already in pom.xml) `Assertions.assertThat(GoTimeUtil.durationSeconds("23s")).isEqualTo(23);/*...*/`
- Use hamcrest assertions `assertThat(GoTimeUtil.durationSeconds("23s"), is(23));/*...*/`